libexec/rtld-elf/Makefile

```diff
@@ -30,6 +30,8 @@ MK_UBSAN=	no
 
 .include <bsd.compat.pre.mk>
 
+RTLD_SANDBOX=
```

Comment:
This clearly shouldn't be set for non-purecap

Reply:
Also this stops libmap.conf being installed for the native ld-elf

Reply:
> Also this stops libmap.conf being installed for the native ld-elf

Hmmm, was there any reason for not installing libmap.conf when RTLD_SANDBOX is set? What's the best thing to do here. Can I just delete the `&& !defined(RTLD_SANDBOX)` bit?

Reply:
Because then rtld-elf and rtld-elf-c18n install duplicate files

Reply:
If you want to keep rtld-elf-c18n around, you need to decouple "is rtld-elf-c18n" from "RTLD-based compartmentalisation code is compiled in".